llarp/iwp/message_buffer.cpp

```diff
@@ -13,6 +13,7 @@ namespace llarp
         : m_Data{std::move(msg)}
         , m_MsgID{msgid}
         , m_Completed{handler}
+        , m_LastFlush{now}
```

Comment:
The net effect of this seems to be that we delay transmission. IIRC, this was even reflected in measurable latency in testing. Should we revisit this?